### PR TITLE
Have code avoid using the "single-statement return exception"

### PR DESCRIPTION
**Pull request by bradcray**
Historically, Chapel has permitted a special case for one-statement procedure bodies that happen to be 'return's.  We are moving away from supporting this exception, so this PR changes the two instances relying on it in Arkouda to use curly brackets instead.

### CODE REVIEW

**Comment by pierce314159** [comment]
hey @bradcray is there still work to be done on this or is it ready for review?

**Comment by pierce314159** [comment]
> @pierce314159 : Sorry for the lack of further information or action here—I was out of the office last week, and worried I'd forget about this branch if I didn't make a PR for it the day I wrote it, but haven't had time to make an issue for it yet (and also haven't merged the change to the Chapel repo that led me to make this branch). Left to my own devices, I'd probably:
> 
> * merge the PR to the Chapel repo (hopefully this week)
> * change this PR to non-draft
> * open an issue describing the reason for the change
>   All that said, this is a completely simple, backwards-compatible change, so could be merged at any time, particularly if you prefer the style of always using curly brackets for procedure bodies.  If you'd rather merge it before I make the Chapel change, let me know and I can create the issue and make it non-draft.

Okay sweet! Thanks for the context! I personally prefer the curly bracket style, and for something so small I don't think an issue is needed. So I'm down to go ahead and review it

**Review by pierce314159 — APPROVED** [review]
This looks good to me! After a quick chat with @Ethan-DeBandi99, I think we prob won't merge this until after we cut our release, but I wanted to go ahead and review since i'll be out for a bit

**Review by Ethan-DeBandi99 — APPROVED** [review]
Looks good

**Pull request merged**
